uikit/src/main/java/com/midtrans/sdk/uikit/activities/CreditCardFlowActivity.java

```diff
@@ -867,4 +881,38 @@ private void showBankPointsFragment(float balance, String bankType) {
         getSupportFragmentManager().beginTransaction().replace(R.id.card_container, fragment).addToBackStack("").commit();
     }
 
+    public boolean isClickPayment() {
```

Comment:
it just want to make sure to support backward compatible as much as possible to prior configuration. so this changes doen't break the old configuratio of current merchant. But later in version 2 we will not do it again